.eslintrc

```diff
@@ -3,5 +3,10 @@
   "extends": ["cozy-app/react"],
   "globals": {
     "jest": true
+  },
+  "settings": {
+    "react": {
+      "version": "15.6.2", // React version, default to the latest React stable release
```

Comment:
Why not `16.5.2` ? Actually it seems you inverted the `5` and the `6`.

Reply:
Thanks, it is 16.2, I had it before upgrading to 16, and cozy-ui actually needs the 16 for the Portal to works. 16.2 and not 16.3 not to have the lifecycle hooks deprecations (componentDidUpdate -> UNSAFE_componentDidUpdate in 16.3)

Reply:
:+1: